pkg/virt-controller/watch/migration.go

```diff
@@ -55,12 +50,9 @@ func (md *MigrationDispatch) Execute(store cache.Store, queue workqueue.RateLimi
 	}
 
 	// Fetch the latest Migration state from cache
-	obj, exists, err := store.GetByKey(key.(string))
+	// error is always nil
+	obj, exists, _ := store.GetByKey(key.(string))
```

Comment:
Again, I would expect the error although, the current used store does not throw one ...